### ISSUE TITLE
Peer review: Add one more choice to the list

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#4797]
peer review feature:

No FoP in copyright-related reasons for deletion (the second question). This is one of the most common reason for deletion, it should be in the list. Proposed message: "There is no freedom of panorama in the country where this picture was taken, so some installations (even permanently displayed in public spaces) can not be freely photographed and shared."

**Comment by strainu** [comment]
![Screenshot_20220206-101156](https://user-images.githubusercontent.com/1571400/152672679-1c694eec-8a43-42ed-8152-77ee6de3f5d5.jpg)

Here is the screen where I believe "No FoP" should appear. Considering the size, I believe something like "Breach of Freedom of Panorama rules" would be OK as a text. If you can add links, you cen link Freedom of Panorama to C:FOP

**Comment by strainu** [comment]
And [here](https://commons.wikimedia.org/w/index.php?title=Commons:Deletion_requests/File:Hexagone-Balard_@_Paris_(23766157612).jpg&oldid=627919512) is the incomplete deletion request. It reads "Because it is ~~~~" in Romanian. Maybe you should use the non-localized version for deletion proposals?

**Comment by nicolas-raoul** [comment]
Thanks! I created more issues based on your feedback, feel free to subscribe to them too and add more information:
https://github.com/commons-app/apps-android-commons/issues/4800
https://github.com/commons-app/apps-android-commons/issues/4801

**Comment by Rishavgupta12345** [comment]
I would like to work on this but I am not able to understand this issue properly. Can you please elaborate and tell me the meaning of FoP

**Comment by strainu** [comment]
@Rishavgupta12345 you have a more detailed explanation below the picture. You need to add another entry in the list, with the text "Breach of Freedom of Panorama rules" and do not create the request If no option is chosen.

Given that I expect other people to ask for other options, maybe using a dropdown instead of checkboxes would be a good idea? @nicolas-raoul any thoughts on replacing the checkboxes with a dropdown? I'm not even sure what happens if *more* options are selected.

**Comment by nicolas-raoul** [comment]
Maybe just add another checkbox for now? If we run into space problem, then switching to a dropdown can definitely be considered (as another GitHub issue).

In parallel, it would indeed be good to test what happens when selecting several options, and if the result is bad then propose a change as another issue. Thanks again! :-)

**Comment by Rishavgupta12345** [comment]
I would like to work on this issue please assign me @nicolas-raoul

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul I have a doubt . whenever I am running the app from android studio its running the beta version of the app. but I want to run the main version of the app in my emulator , how should i do it

**Comment by nicolas-raoul** [comment]
Since this issue is more complicated than expected, I split the second part to its own issue: #4844

**Comment by u7253519** [comment]
Hi I was trying to work on #4801 and I saw that it`s linked to this one. It seemed that the other one can`t be solved with this one finished first, so I tried to work on this one instead. I`ve made a pull request for this issue, can you help me check if it would work alright? I will try to fix remaining problems. Thank you! @nicolas-raoul

**Comment by awanishyadav967** [comment]
As I see this is still open ,I would like to work on this .Please assigned it to me @nicolas-raoul

**Comment by awanishyadav967** [comment]
As I see ,this is still open and I want to work on this .Please assigned me @nicolas-raoul .

**Comment by nicolas-raoul** [comment]
@awanishyadav967 Please use the prodDebug build flavor.

**Comment by awanishyadav967** [comment]
@nicolas-raoul ,Done ,created new pull request please review it.Thanks
